crates/proof-of-sql/src/proof_primitive/dory/blitzar_metadata_table.rs

```diff
@@ -179,10 +188,11 @@ pub fn create_blitzar_metadata_tables(
         / single_entry_in_blitzar_output_bit_table.len())
         .flat_map(|i| {
             itertools::repeat_n(
-                full_width_of_row(i) as u32,
+                u32::try_from(full_width_of_row(i)),
```

Comment:
NIT:
I'd rather expect and panic here.
```suggestion
                u32::try_from(full_width_of_row(i)).expect("row lengths should never exceed u32::MAX"),
```